src/OrchardCore.Modules/OrchardCore.Lucene/Services/LuceneIndexManager.cs

```diff
@@ -54,10 +59,19 @@ LuceneIndexSettingsService luceneIndexSettingsService
                 shellOptions.Value.ShellsApplicationDataPath,
                 shellOptions.Value.ShellsContainerName,
                 shellSettings.Name, "Lucene");
-
             Directory.CreateDirectory(_rootPath);
             _luceneAnalyzerManager = luceneAnalyzerManager;
             _luceneIndexSettingsService = luceneIndexSettingsService;
+
+            // Typical geospatial context
+            // These can also be constructed from SpatialContextFactory
+            _ctx = SpatialContext.GEO;
+
+            var maxLevels = 11; // Results in sub-meter precision for geohash
```

Comment:
I need to change this to be a Lucene Spatial setting.

Reply:
I'm thinking that maybe it could be configurable on each index.